builds.sh

```diff
@@ -19,13 +19,26 @@ shell="$(jq -r '
 ' "$json")"
 eval "$shell"
 
+_resolveRemoteArch() {
+	local img="$1"; shift
+
+	local arches
+	if ! arches="$(bashbrew remote arches --json "$img" 2>/dev/null)"; then # TODO somehow differentiate errors like 404 / 403, "insufficient_scope" from other bashbrew errors here so we can stop eating stderr
+		return
```

Comment:
(this would also make the empty vs not set concerns go away and we could use the `:` forms consistently and be less likely to break the logic in future changes)

Reply:
+1 to both

Reply:
Not sure why it doesn't update here but I added `printf null`